yt/geometry/particle_oct_container.pyx

```diff
@@ -18,7 +18,7 @@ from oct_container cimport OctreeContainer, Oct, OctInfo, ORDER_MAX, \
     SparseOctreeContainer, OctKey, OctAllocationContainer
 cimport oct_visitors
 from oct_visitors cimport cind, OctVisitor
-from libc.stdlib cimport malloc, free, qsort
+from libc.stdlib cimport malloc, realloc, free, qsort
```

Comment:
I *think* we don't use `realloc`

Reply:
oops, I added this before sending the memoryview refactor to Nathan. I was using it to experiment